python/ql/src/semmle/python/types/ModuleObject.qll

```diff
@@ -257,3 +257,12 @@ class PackageObject extends ModuleObject {
 
 }
 
+/** Utility module for predicates relevant to the `ModuleObject` class. */
+module ModuleObject {
+
+    /** Gets a `ModuleObject` who's name is `name` */
```

Comment:
Typo: `whose`. Maybe writing ``called `name` `` instead would be a less awkward construction. Might also want to add `if it exists`.